torch/_dynamo/variables/tensor.py

```diff
@@ -182,7 +182,8 @@ def var_getattr(self, tx, name):
         # It's hard to get resize_() on graph input work properly across
         # dynamo/aot/inductor, just fall back.
         if name in ("resize_", "unsqueeze_") and self.source is not None:
```

Comment:
can you use the [inplace tag](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/tags.yaml#L3-L5) list ? `resize_` and `unsqueeze` is an incomplete list

Reply:
This list should only be for metadata mutation, not all mutation.

Reply:
Yea, sorry I linked to `inplace_vew`, which is metadata only. mistyped